### PR TITLE
99 display metadata in entities app

### LINKED ISSUE #99: Display metadata in Entities app
Metadata of entities will sometimes be created automatically or by the users. An example looks like this:
```JSON
    {
        "id": "TemperatureSensor:device:001",
        "type": "TemperatureSensor",
        "TimeInstant": {
            "type": "DateTime",
            "value": "2022-12-12T10:24:26.495Z",
            "metadata": {}
        },
        "temperature": {
            "type": "string",
            "value": 19.2,
            "metadata": {
                "TimeInstant": {
                    "type": "DateTime",
                    "value": "2022-12-12T10:24:26.495Z"
                }
            }
        }
    }
```
Right now, entities app can not display such an attribute `metadata`

### PR DESCRIPTION
**Pull request by sbanoeon**

### CODE REVIEW

**Comment by djs0109** [comment]
LGTM! @sbanoeon Could you provide a button to beautify the metadata?

**Comment by sbanoeon** [comment]
@djs0109 Currently, the json beautifies when user focuses on the metadata field. Do you think there should be a separate button for it?

**Comment by djs0109** [comment]
@sbanoeon I overlooked that. I think this feature is pretty cool!

File: app/Entirety/entities/views.py [hunk 1]
```diff
@@ -141,14 +141,15 @@ def post(self, request, *args, **kwargs):
...
-            while i < (len(entity_keys) / 3):
+            while i < (len(entity_keys) / 4):
                 keys = [
                     k
                     for k, v in self.request.POST.items()
                     if k in entity_keys and re.search(j.__str__(), k)
                 ]
                 if any(keys):
                     attr = ContextAttribute()
+                    attr.metadata = json.load(self.request.POST.get(keys[3]))
```
Comment (djs0109) [inline app/Entirety/entities/views.py]: Should it be json.loads() here?

File: app/Entirety/entities/views.py [hunk 2]
```diff
@@ -278,14 +288,15 @@ def post(self, request, *args, **kwargs):
...
-        while i < (len(entity_keys) / 3):
+        while i < (len(entity_keys) / 4):
             new_keys = [
                 k
                 for k, v in self.request.POST.items()
                 if k in entity_keys and re.search(i.__str__(), k)
             ]
             if any(new_keys):
                 attr = ContextAttribute()
+                attr.metadata = json.loads(self.request.POST.get(new_keys[3]))
```
Comment (djs0109) [inline app/Entirety/entities/views.py]: Please add error handling for Validation Error

File: app/Entirety/entities/views.py (hunk 1 above)
Comment (djs0109) [reply app/Entirety/entities/views.py]: Would that increase the stability, when the users delete the default value of meta data? The same for update.
```python
attr.metadata = json.load(self.request.POST.get(keys[3])) \
     if self.request.POST.get(keys[3]) else {}
```

**Review by djs0109 — CHANGES_REQUESTED** [review]

**Review by djs0109 — APPROVED** [review]

**Pull request merged**